src/model/schemas.py

```diff
@@ -27,20 +27,40 @@ class Meta:
         strict = True
 
 
+class MediumCompound(Schema):
+    id = fields.String(required=True)
+    # Note: namespace should match a namespace identifier from miriam.
+    # See https://www.ebi.ac.uk/miriam/main/collections
+    namespace = fields.String(required=True)
```

Comment:
Maybe `Namespace` should be its own schema used in both schemata and do the validation.